lib/src/locale/bnLocale.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:jiffy/src/locale/locale.dart';
+import 'package:jiffy/src/locale/relativeTime.dart';
+import 'package:jiffy/src/enums/startOfWeek.dart';
+
+class BnLocale extends Locale {
+  StartOfWeek strtOfWeek;
+  BnLocale(this.strtOfWeek);
+
+  @override
+  RelativeTime relativeTime() => BnRelativeTime();
+
+  @override
+  List<String>? ordinals() => null;
+
+  @override
+  StartOfWeek startOfWeek() => strtOfWeek;
+}
+
+class BnRelativeTime extends RelativeTime {
+  @override
+  String prefixAgo() => '';
+  @override
+  String prefixFromNow() => '';
+  @override
+  String suffixAgo() => 'আগে';
+  @override
+  String suffixFromNow() => 'পরে';
+  @override
+  String lessThanOneMinute(int seconds) => 'কিছু মুহূর্ত';
+  @override
+  String aboutAMinute(int minutes) => 'এক মিনিট';
+  @override
+  String minutes(int minutes) => '$minutes মিনিট';
```

Comment:
I see you've added `bn` locale to the `lib/src/utils/replace.dart` file to replace all number characters but was never used. Please see the suggested edit example

```suggestion
  String minutes(int minutes) => replaceToLocaleNum('$minutes মিনিট', 'bn');
```